### ISSUE TITLE
Button to update

### ISSUE THREAD

**Issue by aericson** [aericson/aarchup#12]
Moving this discussion here.

From email I received.
```
Hello André,
My idea is to add another command line/config option which will allow to define custom action triggered after button is pressed. Also without this option button will not be shown at all. So basically no extra dependency. User will decide on he or she own which command want to use. It would look like that:

aarchup --aur --loop-time 60 --icon /usr/share/aarchup/archlogo.png --update-command 'xfce4-terminal ...'

So basically it can be easily adjusted to any environment and any GUI or terminal application can be run. I think examples for most popular environments may be added in manual.

Thanks,
Krzysztof
```

**Comment by aericson** [comment]
@gumik feel free to submit a pull request if you get it done.
